english/explore-outside/journey-to-flaming-tower/start-running/start-running.md

```diff
@@ -6,4 +6,7 @@ Do you:
 
 [Keep running toward the tower](keep-running/keep-running.md)
 
-[Turn and confront the presence chasing after you](confront/confront.md)
\ No newline at end of file
+[Turn and confront the presence chasing after you](confront/confront.md)
+
+[Throw yourself backward to check your pursuer with your
+shoulder](shoulder-check/shoulder-check.md)
```

Comment:
It looks like this link is broken. I think you could fix it by making the link `../shoulder-check/shoulder-check.md`, but I'd actually prefer if you moved the `shoulder-check` directory within the `start-running` directory. Then this link should work as-is, and I think it's a better directory structure.